microsoft-edge/extensions-chromium/store-policies/ada-addendum.md

```diff
@@ -13,7 +13,7 @@ ms.date: 02/17/2021
 <!-- ====================================================================== -->
 ## Exhibit G: Terms and Conditions for Microsoft Edge Extensions
 
-These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum") is a supplement to the terms and conditions of the App Developer Agreement, the current version of which can be found at [https://go.microsoft.com/fwlink/p/?LinkID=221922](developer-policies.md).  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement, which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.
```

Comment:
Good to go if we are only adding a link to the app developer agreement. Why do we need a CELA review, not sure?

Reply:
fyi, summary explanation of link fix: Here is the current version of present article, 
https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/store-policies/ada-addendum
click the topmost link in article body.

Actual result: brings up article "Microsoft Edge Add-ons store developer policies"
https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/store-policies/developer-policies

Expected result: brings up document "App Developer Agreement":
https://go.microsoft.com/fwlink/p/?LinkID=221922

Remainder of present article refers to "App Developer Agreement" and its numbered sections; it doesn't refer to article "Microsoft Edge Add-ons store developer policies" as far as I can tell, so the link destination "developer-policies.md" appears to be incorrect.

Reply:
Approved. No need for cela review since we are fixing a bug.